PocketKit/Tests/SyncTests/SpaceTests.swift

```diff
@@ -0,0 +1,68 @@
+import XCTest
+
+@testable import Sync
+import CoreData
+
+class SpaceTests: XCTestCase {
+    var context: NSManagedObjectContext!
+    
+    override func setUp() {
+        context = PersistentContainer.testContainer.viewContext
+    }
+    
+    override func tearDownWithError() throws {
+        try Space.testSpace().clear()
+    }
+    
+    func subject(context: NSManagedObjectContext? = nil) -> Space {
+        Space(context: context ?? self.context)
+    }
+    
+    func testDeletingOrphanTags() throws {
+        let space = subject()
+        let _: Tag = space.new()
+        let _: Tag = space.new()
```

Comment:
I was thinking about this test after we hung up our call on Friday. We should create a tag that _does_ have an associated `SavedItem` and make sure that one doesn't get deleted.

Reply:
good point! will update